groupsforsections.py

```diff
@@ -0,0 +1,85 @@
+import json
+import sys
+import logging
+import requests
+import urllib
+import utils
+import yaml
+
+
+def create_group_category():
+    logging.debug('create_group_category() Called')
+
+    url = canvas_url + '/api/v1/courses/' + course_id + '/group_categories?name=' + urllib.quote_plus(
+            group_category_name)
+    response = api_handler(url, 'Post')
+    if response is None:
+        logging.error('Api call for \'creating group categories\' is not successful')
+        sys.exit(1)
```

Comment:
i personally feel catching (and logging it) it better than throwing an exception to a function that is expecting a value. I know here there is nested function call and it is kind of creating a confusion where am i catching it. I will like to have a chat it get things cleared